src/Data/Nat/DivMod.agda

```diff
@@ -223,6 +228,32 @@ m≥n⇒m/n>0 {m@(suc m-1)} {n@(suc n-1)} m≥n = begin
   n / d + (m * n) / d ≡⟨ cong (n / d +_) (*-/-assoc m d∣n) ⟩
   n / d + m * (n / d) ∎
 
+/-step : ∀ {m n n≢0} → m ≥ n → (m / n) {n≢0} ≡ 1 + ((m ∸ n) / n) {n≢0}
+/-step {zero} {zero} {()} _
+/-step {zero} {suc n} {_} ()
```

Comment:
You can omit these lines as since 2.6.0 Agda is now clever enough to infer that they are impossible.